### PR TITLE
Codeowners

### PR DESCRIPTION
**Pull request by obaranov1**
add codeowners and remove index file from main branch

### CODE REVIEW

**Comment by mythi** [comment]
@obaranov1 `CODEOWNERS` looks OK to me but let's get rid of the unrelated changes caused by the editor(?)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Syntax and details are here: https://help.github.com/articles/about-codeowners/
+#
+# Global code ownership, fallback
+*                   @mythi @poussa @softwarerecipes
```
Comment (poussa) [inline .github/CODEOWNERS:4]: Actually, take mythi and poussa out.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# CI/CD
+.github/workflows/ @mythi @poussa @softwarerecipes
```
Comment (poussa) [inline .github/CODEOWNERS:7]: Delete this line, not needed.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# charts
+/charts/device-plugin-operator/  @mythi @poussa
+/charts/evam-0.7.2/  @softwarerecipes
```
Comment (poussa) [inline .github/CODEOWNERS:11]: remove this line, covered by global.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+/charts/gpu-device-plugin/  @mythi @poussa
+/charts/load-generator-0.1.2/  @softwarerecipes
```
Comment (poussa) [inline .github/CODEOWNERS:13]: same here

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+# CI/CD
+.github/workflows/ @mythi @poussa @softwarerecipes
+
+# charts
+/charts/device-plugin-operator/  @mythi @poussa
+/charts/evam-0.7.2/  @softwarerecipes
+/charts/gpu-device-plugin/  @mythi @poussa
+/charts/load-generator-0.1.2/  @softwarerecipes 
+/charts/multimodal-data-visualization-3.0.0/  @softwarerecipes
```
Comment (poussa) [inline .github/CODEOWNERS:14]: and here

**Review by poussa — CHANGES_REQUESTED** [review]
Also, remove the extra files as @mythi suggested.

File: .github/CODEOWNERS (hunk 3 above)
Comment (obaranov1) [reply .github/CODEOWNERS:11]: hey, what exactly should be removed?

Comment (mythi) [reply .github/CODEOWNERS:11]: the charts that are not related to device plugins (evam, load-generator, multimodal...)

**Pull request closed without merging**